Cargo.toml

```diff
@@ -16,7 +16,7 @@ ansi_term = "0.12.1"
 anyhow = "1.0.32"
 atty = "0.2.14"
 clap = "3.0.0-beta.1"
-parsec-client = "0.10.0"
+parsec-client = { git = "https://github.com/joechrisellis/parsec-client-rust.git", branch = "add-list-keys" }
```

Comment:
Now that the client is merged, you can use that specific commit from `parallaxsecond/parsec-client-rust`!

Reply:
Good call, thanks. 😄